### ISSUE TITLE
Check Goude flow curvature correction

### ISSUE THREAD

**Issue by petebachant** [turbinesFoam/turbinesFoam#230]
>I think there is a sign error on the line in actuatorlineelement.C
```c++
angleOfAttackRad += omega_*(chordMount_ - 0.25) * chordLength_/mag(relativeVelocity_);
```
>and that it really should be
```c++
angleOfAttackRad += omega_*(chordMount_ + 0.25) * chordLength_/mag(relativeVelocity_);
```

>The original model was designed with `chordMount = 0` in the middle of the blade (i.e. 0.25 being the quarter chord position). `turbinesFoam` uses `chordMount=0` as `quarterchord`, hence, it should be +0.25 to compensate for this difference in definition of `chordMount`.

**Comment by petebachant** [comment]
>In the derivation in my thesis, 0 is used as mid chord and 0.25 is used as quarterchord and -0.25 as threequarterchord, i.e. with the opposite sign. When using midchord position, the first term in the correction should be zero. When using the threequarterchord position, the entire goude correction (both terms combined) should be zero. Note also that the angle of attack also is defined with the opposite sign in my thesis, as I used negative angles of attack on the upwind side. Hence, the sign error may be on the entire first term of the correction.

**Comment by petebachant** [comment]
Should also take a look at http://www.sciencedirect.com/science/article/pii/S0196890415011656

**Comment by petebachant** [comment]
Looks like when switching sign, we have a huge negative power coefficient:

![image](https://user-images.githubusercontent.com/4604869/28239860-94bbd026-6943-11e7-9324-dec0cd1252ad.png)

Change to angle of attack is:

![image](https://user-images.githubusercontent.com/4604869/28239864-a801773a-6943-11e7-9691-2f64508eb844.png)

**Comment by petebachant** [comment]
Also note the end effects model was turned off, else C_P would be extremely low.

**Comment by petebachant** [comment]
Wondering if maybe we need to artificially crank up the Reynolds number to get around the questionable foil data at low Re, which should be okay since the experiments were Re independent.

**Comment by andersgoude** [comment]
I would say that the correction is not valid for those huge corrected values. We are using airfoil data for a symmetric airfoil and assuming that the same data still works for the curved flow if we simply correct the angle of attack. When we have corrections up to 20 degrees, I find it questionable that the simplified way of using symmetric foil data still works.
In general, the dynamic stall models do not work very well at these high angles of attack either. The low tip speed ratio case is always hard for any simulation model based on a blade force model.

**Comment by petebachant** [comment]
I agree. Some have come up with transformations for entire foil datasets based on flow curvature, correct? That would be the ideal solution. Either way, expecting perfect performance predictions for such a high solidity turbine is probably unrealistic.

**Comment by andersgoude** [comment]
Yes, one would need some improved corrections for the foil datasets for these cases. I am not aware of any already existing though, and I would expect that there would be a lot of work to develop it.
